### PR TITLE
Migrate from Ava to Jest

### PR DESCRIPTION
**Pull request by rkrisztian**
[Migrate from Ava to Jest](https://github.com/adnovum/gocd-cypress/pull/5/commits/55c33dfc50f686b3313446d820c160e573e2d1d0)

AVA is a highly opinionated, minimalist testing framework, so it misses some features like test grouping and code coverage, and
it does not use globals, which would normally be a good thing, but the API also increases verbosity. At the same time Jest is more
popular, and it has more readable assertions. It is true that it does not support custom failure messages, but it also prints
the failing code lines, which is enough considering the code should also be readable.

[Fix ESLint configuration so it works for the test project](https://github.com/adnovum/gocd-cypress/pull/5/commits/db324bdd42298992872e01fbe8b0b39fb5a00e5d)

### CODE REVIEW

**Review by dbalakirev — COMMENTED** [review]
I cannot contribute any opinions, just a LGTM

File: .eslintignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
-node_modules
```
Comment (nemeth-z) [inline .eslintignore:1]: why did you remove these two entries?

Comment (rkrisztian) [reply .eslintignore:1]: * `node_modules` is ignored by default:
https://eslint.org/docs/user-guide/configuring/ignoring-code#the-eslintignore-file
* I did not remove the ignore for `.eslintrc.js`, just extended it to match all of those files.

**Review by dodie — APPROVED** [review]
👍

**Review by nemeth-z — APPROVED** [review]

**Review by hosszubalazs — APPROVED** [review]
